types/hits/samples.go

```diff
@@ -15,20 +15,51 @@ var SampleHits = 99900.
 
 func Samples() error {
 	log.Infoln("Inserting Sample Service Hits...")
-	for i := int64(1); i <= 5; i++ {
-		records := createHitsAt(i)
-		if err := gormbulk.BulkInsert(db.GormDB(), records, db.ChunkSize()); err != nil {
-			log.Error(err)
-			return err
-		}
+	records_1 := createHitsAt(1, -90 * types.Day, 1 * time.Hour)
```

Comment:
aha, I went through the postman tests and figured out this is what's "wrong":

It wants at least 100 hits..
https://github.com/statping-ng/statping-ng/blob/4d872facdafa1f527d5b2c46e4c09623804f5e2c/dev/postman.json#L1479-L1481

..in the last three days..
https://github.com/statping-ng/statping-ng/blob/4d872facdafa1f527d5b2c46e4c09623804f5e2c/dev/postman.json#L20-L22

.. which, at 1/h, is not the case (that's where the actual 72 / expected 100 is coming from)

Since this is only checked for the service with id 1, it really doesn't matter where we/you change this -
* bump up the sample hit frequency here (to 42-ish minutes or lower for 100 in 3 days)
* reduce the expected number of hits to 72
* increase the time range the tests looks at (`{{start_time}}` is only used in this one test)

like I said, I don't mind any of those. you pick :b

Reply:
I went through and added more data samples and it works consistently now :D Figured I would add more realistic data with minute intervals.

I was able to run all the tests in postman successfully locally, hopefully this works now 😸